### PR TITLE
chore(chaos-center): Updated hub mountPath and minor fix in deleteAgent mutation

### PR DESCRIPTION
**Pull request by amityt**
Signed-off-by: Amit Kumar Das <amit@chaosnative.com>

## Proposed changes

This PR adds:
- mountPath fix for hub
- minor label fix in subscriber

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by Jonsy13 — APPROVED** [review]

File: litmus-portal/cluster-k8s-manifest.yml [hunk 1]
```diff
@@ -589,7 +589,7 @@ spec:
           volumeMounts:
             - mountPath: /tmp/gitops
               name: gitops-storage
-            - mountPath: /data/tmp
+            - mountPath: /tmp/version
```
Comment (gdsoumya) [inline litmus-portal/cluster-k8s-manifest.yml:592]: probably `/tmp/hub` would be a better name

**Review by gdsoumya — APPROVED** [review]

**Comment by gdsoumya** [comment]
Conflict possible with PR https://github.com/litmuschaos/litmus/pull/3486

File: litmus-portal/graphql-server/pkg/cluster/handler/handler.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package handler
 import (
 	"context"
 	"fmt"
+	"github.com/litmuschaos/litmus/litmus-portal/graphql-server/pkg/authorization"
```
Comment (gdsoumya) [inline litmus-portal/graphql-server/pkg/cluster/handler/handler.go:6]: run goimports

**Review by gdsoumya — CHANGES_REQUESTED** [review]

File: litmus-portal/cluster-k8s-manifest.yml (hunk 1 above)
Comment (amityt) [reply litmus-portal/cluster-k8s-manifest.yml:592]: @gdsoumya We are planning to add a separate env for chaoshub paths. Will take this in the refactoring PR.

**Review by gdsoumya — APPROVED** [review]

**Review by rajdas98 — APPROVED** [review]

**Pull request merged**